nginx.conf

```
@@ -0,0 +1,56 @@
+user                            nginx;
+worker_processes                1;
+
+error_log                       /var/log/nginx/error.log warn;
+pid                             /tmp/nginx.pid;
+
+events {
+  worker_connections            1024;
+}
+
+http {
+  include                       /etc/nginx/mime.types;
+  default_type                  application/octet-stream;
+  charset                       UTF-8;
+  sendfile                      on;
+  log_format                    main '$remote_addr - $remote_user [$time_local] "$request" '
+                                     '$status $body_bytes_sent "$http_referer" '
+                                     '"$http_user_agent" "$http_x_forwarded_for"';
+  client_body_temp_path         /tmp/client_temp;
+  proxy_temp_path               /tmp/proxy_temp_path;
+  fastcgi_temp_path             /tmp/fastcgi_temp;
+  uwsgi_temp_path               /tmp/uwsgi_temp;
+  scgi_temp_path                /tmp/scgi_temp;
+  access_log                    /var/log/nginx/access.log main;
+
+  server {
+    include                     /usr/share/nginx/html/headers;
+    listen                      8080;
+    server_name                 localhost;
+    client_max_body_size        32m;
+
+    gzip                        on;
+    gzip_proxied                expired no-cache no-store private auth;
+    gzip_types                  text/plain application/x-javascript text/xml text/css application/xml;
+
+    root                        /usr/share/nginx/html/insights;
```

Shouldn't it be `catalog` here?

```suggestion
    root                        /usr/share/nginx/html/catalog;
```